src/renderers/webgl/WebGLTextures.js

```diff
@@ -100,10 +100,13 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
-	function getInternalFormat( glFormat, glType ) {
+	function deduceInternalFormat( textureInternalFormat, glFormat, glType ) {
```

Comment:
I think I would keep the name `getInternalFormat()`. The term `deduce` feels a bit uncommon for the library.

Reply:
Yes but `getInternalFormat` taking itself as an argument the internalFormat was a bit strange. Up to you!